chapters/annotations.tex

```diff
@@ -1349,15 +1381,16 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 \begin{nonnormative}
 When converting between units (in the user-interface for plotting and entering parameters), the unit offset must be ignored for a variable defined with annotation \lstinline!absoluteValue = false!.
 This annotation is used in the Modelica Standard Library, for example in \lstinline!Modelica.Units.SI! for the type definition \lstinline!TemperatureDifference!.
+For most types there is no unit offset and the annotation is not needed for them.
```

Comment:
We could add that "For types with a unit offset it is recommend to have the annotation for clarity." But I believe that should be a separate PR.